src/common/hooks/use-tx-state.ts

```diff
@@ -25,11 +25,11 @@ export const useTxState = () => {
   const signedTransaction = useLoadable(signedTransactionStore);
   const isUnauthorizedTransaction = useRecoilValue(isUnauthorizedTransactionStore);
 
-  const doSubmitPendingTransaction = useRecoilCallback(
+  const doSubmitOrCancelPendingTransaction = useRecoilCallback(
```

Comment:
I think we should break these out into two methods, `doSubmitPendingTransaction` and `doCancelPendingTransaction`. I am also open to changing the convention we use for naming methods. I have a preference for `handleX`, but curious what you both think @fbwoolf @kyranjamie

Reply:
Agree on breaking into multiple methods

Reply:
Yep, I agree, I was definitely intending to split these up. I typically have used `handleX` in the past. I have never used `do` before.

Reply:
`do` is my fault haha, it's something I was really into for a while when I was using redux and the conventions found here https://reduxbundler.com/ and sold Hank on it, and he has used it ever since

Reply:
I'd like to move away from this convention ideally. It isn't followed exactly. Do _all_ functions begin with do? It's bad English. Somewhat unrelated, but saw [this opinion from above package author](https://twitter.com/HenrikJoreteg/status/1393061583484383235) which is pretty cringe

Reply:
yeah i mean i no longer use the convention, and happy to move away from it in our codebase :) but let's keep that kind of refactor to a new PR, but please use `handleX` for any new methods moving forwar